client/src/actions/register.coffee

```diff
@@ -0,0 +1,25 @@
+{Promise} = require 'es6-promise'
+Jwt = require 'jwt-simple'
+
+types = require '../constants/action_types.coffee'
+
+{getRepos} = require './repo.coffee'
+
+RegisterActions =
+  register: (username, password) ->
+    (dispatch) ->
+      payload =
+        user: 'test-user'
+        role: 'user'
+
+      registerResult = Promise.resolve({
+        type: types.REGISTER
```

Comment:
This probably should be REGISTER_SUCCESS. That correctly implies that registration is done, and it permits an event REGISTER_FAILED to exist when the backend is hooked up.